src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -342,11 +346,13 @@ trait ClusterComponent extends LeoComponent {
     }
   }
 
+
   // select * from cluster c left join label l on c.id = l.clusterId
-  val clusterQueryWithLabels: Query[(ClusterTable, Rep[Option[LabelTable]]), (ClusterRecord, Option[LabelRecord]), Seq] = {
+  val clusterQueryWithLabels: Query[(ClusterTable, Rep[Option[LabelTable]], Rep[Option[ClusterErrorTable]]), (ClusterRecord, Option[LabelRecord], Option[ClusterErrorRecord]), Seq] = {
```

Comment:
Something I've been thinking about: maybe the `list*` methods should ONLY join to labels, and the `get*` methods should join to all supporting tables: errors and (soon) instances. That way the list APIs are cheaper and return a stripped-down version of each cluster, while the "get details" API returns all the information about the cluster.